packages/react/src/components/DetailsList/index.ts

```diff
@@ -2,18 +2,25 @@ export * from '../../Selection';
 export * from '../GroupedList/GroupedList.types';
 export * from './DetailsHeader';
 export * from './DetailsHeader.base';
+export * from './DetailsHeader.styles';
```

Comment:
Because DetailsRow exported styles, I exported the other styles for consistency and to ensure callers could use those style functions if they want to extend the base class with styled<>.  I had to rename the getStyles calls to be specific to the component to avoid name collisions.

Reply:
Should these also have _unstable? like `useTextareaStyles_unstable`

Reply:
No - that is a v9 thing.  These are all stable.